contents/docs/user-guides/groups.md

```diff
@@ -30,53 +43,51 @@ As another example, if `transaction` is a group type, then each `transactionID`
 
 These properties can be used in insights similar to how you use person properties.
+
 ## How to think about Insights on groups
 
 Since there's no new events, you can use the same events you've been using for your insights. It's the same funnel you already have, aggregated differently.
 
-The groups determine what your insights aggregate on. For example, consider the funnel with two events: `User signed up -> User did something important` (tk: better example?)
+The groups determine what your insights aggregate on. For example, consider the funnel with two events: `User signed up -> User activated`
 
-Traditionally, you'd only have unique users in the funnel: Users from any and every company who're using your product and doing something important. Your conversion rate is per user. But what if you're interested in looking at this conversion rate across companies? Say, you're a B2B product and doing something important once per company is good enough. Then the funnel you want to track is `User signed up -> User did something important` aggregated by unique companies, not users.
+Traditionally, you'd only have unique users in the funnel: Users from any and every company who're using your product and doing something important. Your conversion rate is per user. But what if you're interested in looking at this conversion rate across companies? Say, you're a B2B product and doing something important once per company is good enough. Then the funnel you want to track is `User signed up -> User activated` aggregated by unique companies, not users.
 
 That's a group aggregation over the company group type. It's the same events, aggregated differently.
 
 What if you wanted to breakdown this funnel by company name? Maybe there's specific companies that never converted, and you want to follow up with them to see what's going wrong. Like mentioned above, if you have group properties defined, you can breakdown by `company_name`. 
 
-TODO: consistent capitalisation
-
 ## How to think about Feature Flags with groups
 
-Similar to insights, where you're aggregating events by group type, you can have Feature Flags that work on groups. This allows you to rollout a feature by company, instead of users.
+Similar to insights, where you're aggregating events by group type, you can have Feature Flags that work on groups. This allows you to rollout a feature by company, instead of users. Preventing disruption when two users at the same company see a different experience.
 
-With group properties defined, you can even do things like rolling out a feature to companies where the `user_count` is less than, say, 3. Maybe it's a feature for small teams, or maybe you simply want to minimise impact in the beginning.
+With group properties defined, you can even do things like rolling out a feature to `companies` where the `user_count` is less than, say, 3. Maybe it's a feature for small teams, or maybe you simply want to minimise impact in the beginning.
```

Comment:
This feature doesn't exist. This is a potential aspirational usecase from team growth but not something we're currently building out.

Reply:
Noted - will remove.